drivers/lora/Kconfig.sx12xx

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2019 Manivannan Sadhasivam
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+DT_COMPAT_SX1276 = semtech,sx1276
+DT_COMPAT_SX1261 = semtech,sx1261
+DT_COMPAT_SX1262 = semtech,sx1262
+
+menuconfig LORA_SX12XX
+	bool "Semtech SX-series driver"
+	select HAS_SEMTECH_RADIO_DRIVERS
+	select HAS_SEMTECH_LORAMAC
+	depends on SPI
+	help
+	  Enable LoRa driver for Semtech SX12xx.
+
+if LORA_SX12XX
+
+choice
+	prompt "LoRa Radio chipset"
```

Comment:
Is this really a choice? Is it possible to have two different LoRa radios in one system, one with an SX1276 and another with an SX1261? If so, I think these would be best expressed as separate `bool` options, each of which enables support for building an individual driver. Kind of like the example in https://github.com/zephyrproject-rtos/zephyr/pull/24616#discussion_r420311757.

Reply:
The LoRaMac-node library assumes that there is only one LoRa radio in the system. For that reason, it has been designed using global struct to bind the radio and the MAC. There is no easy way to work around that without extensive hacking of the library.

For boards that have both (I was in fact testing with such a board for a little while), the user would have to select which radio to use at build time.